src/lib/utils/cosmosSdk.ts

```diff
@@ -8,6 +8,10 @@ import { cosmosSdkChainIds } from '@shapeshiftoss/chain-adapters'
 import { KnownChainIds } from '@shapeshiftoss/types'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
 
+export const isCosmosSdkChainId = (chainId: KnownChainIds | ChainId): boolean => {
+  return cosmosSdkChainIds.includes(chainId as CosmosSdkChainId)
+}
```

Comment:
Can check against `fromChainId(chainId).chainNamespace` if you feel like it

Reply:
Not needed anymore as this util has been removed